.github/knip.ts

```diff
@@ -3,9 +3,9 @@ import type { KnipConfig } from "knip";
 const config: KnipConfig = {
   entry: ["src/worker.ts"],
   project: ["src/**/*.ts"],
-  ignoreBinaries: ["build", "i", "format:cspell", "awk", "lsof"],
+  ignoreBinaries: ["i"],
```

Comment:
What is `i`

Reply:
`Knip` recognizes `bun i` as `i` being a command and complains.